docs/development/doc_guide.rst

```diff
@@ -1,17 +1,164 @@
 ************************
-Documentation Guidelines
+Documentation guidelines
 ************************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+PlasmaPy's online documentation is hosted by `Read the Docs`_
+and can be found online at:
+
+* Most recent stable release:
+  `https://docs.plasmapy.org <https://docs.plasmapy.org>`_ or
+  `https://docs.plasmapy.org/en/stable/ <https://docs.plasmapy.org/en/stable/>`_
+
+* Latest version on GitHub:
+  `https://docs.plasmapy.org/en/latest/ <https://docs.plasmapy.org/en/latest/>`_
```

Comment:
Might be useful to define the difference between what is meant by stable and latest, i.e. stable = last release and latest = Head of master.

Reply:
This might also be worth adding in an admonition on the documentation landing page.  I would not be surprised if this is a common point of confusion when users seek out documentation, any documentation not just PlasmaPy.

Reply:
Yeah, an admonition would be really helpful.  I created #1164 for doing this on the landing page.